aws/table_aws_scheduler_schedule.go

```diff
@@ -0,0 +1,241 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/scheduler"
+	"github.com/aws/aws-sdk-go-v2/service/scheduler/types"
+
+	schedulerv1 "github.com/aws/aws-sdk-go/service/scheduler"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsSchedulerSchedule(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_scheduler_schedule",
```

Comment:
```suggestion
		Name:        "aws_eventbridge_schedule",
```
Should we use the above table name?